CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Add `buf-warning-bin` header to detect and print server warning messages.
```

Comment:
I don't know if this is worthy of a changelog update actually

Reply:
Removed